scripts/lib.psm1

```diff
@@ -0,0 +1,172 @@
+#Requires -Version 5
+
+function Get-PlatformInfo {
+    [OutputType([object])]
+    param()
+
+    # Dev guide links
+    $DEV_GUIDE_URL = 'https://mytableau.tableaucorp.com/x/LYIUEw'
+    $LINUX_DEV_URL = 'https://mytableau.tableaucorp.com/x/RaPnH'
+    $MACOS_DEV_URL = 'https://mytableau.tableaucorp.com/x/W1BsH'
+
+    if (-not (Test-Path -Path variable:IsWindows) -or $IsWindows) {
+        [PSCustomObject]@{
+            IsWindows = $true
+            IsMacOS = $false
+            IsLinux = $false
+            RuntimeId = 'win-x64'
+            DevGuideUrl = $DEV_GUIDE_URL
+            PlatformDevGuideUrl = $null
+        }
+    }
+    elseif ($IsMacOS) {
+        [PSCustomObject]@{
+            IsWindows = $false
+            IsMacOS = $true
+            IsLinux = $false
+            RuntimeId = 'osx-x64'
+            DevGuideUrl = $DEV_GUIDE_URL
+            PlatformDevGuideUrl = $MACOS_DEV_URL
+        }
+    }
+    elseif ($IsLinux) {
+        [PSCustomObject]@{
+            IsWindows = $false
+            IsMacOS = $false
+            IsLinux = $true
+            RuntimeId = 'linux-x64'
+            DevGuideUrl = $DEV_GUIDE_URL
+            PlatformDevGuideUrl = $LINUX_DEV_URL
+        }
+    }
+    else {
+        throw 'Get-PlatformInfo: Unsupported platform'
+    }
+}
+
+function Get-DefaultSubmoduleDirs {
```

Comment:
I didn't think this had any sub modules. 
Did you copy/paste this file? If so, can you clean it up so it only has the things you require.